### ISSUE TITLE
navigation doesn't work on next search page

### ISSUE THREAD

**Issue by hex0cter** [indrek-koue/No-Mouse-Google-Search-Result-Navigation#17]
Hi,

The extension works great with Google search on my Macbook. However when I hit Enter on Next and the next page loads, the keyboard navigation doesn't work any more (if I reload the page, the shortkeys will come back). Is this something you are aware of?

I am using Chrome: Version 48.0.2564.116 (64-bit) on OSX El Capitan.

**Comment by hex0cter** [comment]
alright, it seems I misunderstood it. In order to get the navigation, all I need is to hit the tab key!
